### DISCUSSION TITLE
Performance Benchmarking Minio on EBS vs S3 for object storage

### DISCUSSION THREAD

**Discussion by VijayDeyTR** [milvus-io/milvus#21716] [Q&A and General discussion]
Hi!

We would like to deploy Mivlus on AWS using S3 as Object Storage. Are there any benchmarks or performance evaluation results (QPS/Search Latency/Indexing Rate,etc) that compare using Minio with EBS vs S3?

**Comment by xiaofan-luan** [comment]
If it's on AWS, we definitely recommend to use S3 because if will be much cheaper than ebs and more stable.
Cheers!
